cfgov/form_explainer/jinja2/form-explainer/index.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layout-full.html' %}
```

Comment:
@chosak That is a great point! There is a lot of repetition across the templates, so it would be great to standardize some of the content in a single location.  I hate to add to the duplication here, but it seems like the solution might need some discussion -- would it make sense to create an issue and tackle all the templates together?